packages/libraries/core/tests/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "include": ["."],
   "compilerOptions": {
     "baseUrl": ".",
-    "rootDir": ".",
+    "rootDir": "..",
```

Comment:
if rootdir doesnt include imported files, then tsc complains. This is doesn't do anything in reality, but it's annoying in the IDE